cypress/support/commands.ts

```diff
@@ -1,24 +1,40 @@
-import '@percy/cypress';
+Cypress.Commands.add('modalIsVisibleAndFocused', () => {
+  cy.wait(1000);
+  cy.get('.DocSearch-Modal').should('be.visible');
+  cy.get('.DocSearch-Input').should('be.focus');
+});
+
+Cypress.Commands.add('modalIsNotVisible', () => {
+  cy.wait(1000);
+  cy.get('body').should('not.have.class', 'DocSearch--active');
+  cy.get('.DocSearch-Modal').should('not.exist');
+});
 
 Cypress.Commands.add('darkmode', () => {
   cy.get('.react-toggle').click({ force: true });
   cy.get('.react-toggle-screenreader-only').blur();
+  cy.wait(1000);
 });
 
 Cypress.Commands.add('openModal', () => {
   cy.get('.DocSearch-Button').click();
+  cy.wait(1000);
```

Comment:
Wouldn't it be more reliable to wait a .class? (apply for all the new wait())

Reply:
good point, done here: https://github.com/algolia/docsearch/pull/1385/commits/2fddd3591504f0da0ff2090942c13734b59d96f2